dartsim/src/SDFFeatures.cc

```diff
@@ -1115,30 +1101,6 @@ Identity SDFFeatures::ConstructSdfJoint(
   joint->setTransformFromParentBodyNode(parent_T_prejoint_init);
   joint->setTransformFromChildBodyNode(child_T_postjoint);
```

Comment:
Is this block up here still needed? The comment seems outdated

Reply:
agreed, the comment seems outdated

Reply:
I'm going to merge this now to unbreak things but will open an issue to follow up on this comment

Reply:
Yes, the code is needed, but the comment is indeed outdated. I think it can simply be removed.

Reply:
https://github.com/ignitionrobotics/ign-physics/pull/280